### ISSUE TITLE
Sort issue with some movies e.g Die Hard.

### ISSUE THREAD

**Issue by gadgetchnnel** [boxeehacks/boxeehack#310]
It seems that the option to ignore prefixes ("the","a", etc.) is behaving in unexpected ways for some movies in this version (I didn't notice this in previous versions).
Movies which start with an English word which is the same as a prefix in another language are sorted based on the second word in the title. For example, Die Hard appears under H.

I presume this is due to the advancedsettings.xml file which gets copied by the hack, including all the prefixes in the **sorttokens** element (which presumably in regular Boxee are probably set based on the language settings).

I don't know the best way of handling this, maybe there should be a language setting in Boxee+Hacks and multiple advancedsettings.xml files, one per language.

**Comment by boxeehacks** [comment]
Unfortunately that doesn't make much sense. It's the way Boxee solved it themselves, but it's a crappy way. What if a German user has the movie Die Hard in their list... It'll still sort it this wrong way. And if I am a normally English speaking user, but I have a French movie in my list, it'll also get sorted wrong.

The proper way to do this would be to have NFOs which contain both a title and a sort-title. But I'm not sure how we can get Boxee to pick up on this properly (if it is possible). Will look into this...

**Comment by gadgetchnnel** [comment]
Boxee+Hacks must be having some effect on this, as I just uninstalled it to test and the two two movies affected for me - Die Hard 4.0 (I used an NFO to force that title as I hate the US title "Live Free or Die Hard") and Die Another Day - both appear under "D" instead of "H".

I suspect that the default value for **sorttokens** if there is no advancedsettings.xml file is based on the language, but the file copied by the hack overrides that and causes this issue.

**Comment by boxeehacks** [comment]
Yes, but if you set your boxee box language to german you still get the same bug as explained above. The hack override improves things a lot for users that have many foreign films (as I do). In regular boxee if you have both french, german and english language films the default sorttokens are useless.
